client/src/components/Project/index.jsx

```diff
@@ -17,6 +18,13 @@ class Scrums extends Component {
   }
 
   componentDidMount() {
+    const projectId = this.props.match.params.projectId;
+    const scrumId = this.props.match.params.scrumId;
+    if (Number.isInteger(parseInt(projectId)) && Number.isInteger(parseInt(scrumId))){
```

Comment:
I don't understand what is the purpose of this `if` statement :thinking:

Reply:
That's to make sure the scrumId is a number, not a string. without this validation, the app will crash if you tried to fetch a URL like `/projects/1/israa`

Reply:
I don't think it would crash, it'd just return an empty array